test.html

```diff
@@ -32,6 +34,6 @@
         println(mouseX, mouseY);
       }
     </script>
-    <canvas></canvas>
+    <canvas id="canvas"></canvas>
```

Comment:
this is not necessary. Without a target, Processing.js ties code it sees to the closest sibling `<canvas>` element